src/HieDb/Query.hs

```diff
@@ -158,9 +158,10 @@ declRefs db = do
     "refs.dot"
     ( export
         ( ( defaultStyle ( \( _, hie, occ, _, _, _, _ ) -> hie <> ":" <> occ ) )
-          { vertexAttributes = \( mod', _, v : occ, _, _, _, _ ) ->
```

Comment:
This was partial pattern match. Although this pattern match would always succeed, because we always store namespace Char as part of each OccName in DB, it can be avoided without too much extra code.